### PR TITLE
text_11

### PR DESCRIPTION
**Pull request by erinlincoln**
- Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/text_11.ncl
 - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/text_11_lg.png
 - Issue #344 
 
image

### CODE REVIEW

**Review by anissa111 — COMMENTED** [review]
Haha, this is a funny issue. When I plot locally and when I run the html, the plot is showing up with a title?

image

I know you aren't setting this manually, and I suspect it's coming from plotting with xarray on line 44. Why this isn't overwritten later when you specify an empty string for `maintitle`, I'm not sure. 

Since it isn't showing up on the plot you pasted in the comment, I think that this is something that spyder is doing differently than pycharm or the html generation. If you need any help debugging, let me know!

**Comment by erinlincoln** [comment]
I think I turned off the title - I checked it locally on my computer, so I think it is good!

File: Plots/Line/NCL_text_11.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+"""
+NCL_text_11.py
+==============
+This script illustrates the following concepts:
+    - Labelling both Y axes
```
Comment (michaelavs) [inline Plots/Line/NCL_text_11.py:5]:
```suggestion
    - Labeling both Y axes
```

File: Plots/Line/NCL_text_11.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    - Chaning padding for axis labels
```
Comment (michaelavs) [inline Plots/Line/NCL_text_11.py:6]:
```suggestion
    - Changing padding for axis labels
```

**Review by michaelavs — COMMENTED** [review]
Just two minor spelling issues to be updated, but everything else looks good on my side!

**Review by anissa111 — APPROVED** [review]
This looks good to me, the title is no longer showing up on my end!

**Pull request merged**